projects/sandbox/pyproject.toml

```diff
@@ -87,35 +93,72 @@ valid_stride = 0.0625
 commands.maf = "${base.maf}"
 
 
-[tool.typeo.scripts.generate-testing]
+[tool.typeo.scripts.generate-pp-plot-injections]
 ifos = "${base.ifos}"
 state_flag = "${base.state_flag}"
 frame_type = "${base.frame_type}"
 channel = "${base.channel}"
 start = "${base.stop}"
 stop = "${base.test_stop}" 
 sample_rate = "${base.sample_rate}"
-prior = "mlpe.injection.priors.sg_uniform"
+prior = "${base.prior}"
 waveform = "mlpe.injection.waveforms.sine_gaussian_frequency"
 n_samples = 1000
 kernel_length = "${base.kernel_length}"
 waveform_duration = "${base.waveform_duration}"
 datadir = "${base.datadir}"
 logdir = "${base.logdir}"
+gaussian = false
+
+[tool.typeo.scripts.generate-bilby-injections]
+bilby_duration = "${base.bilby_duration}"
+ifos = "${base.ifos}"
+state_flag = "${base.state_flag}"
+frame_type = "${base.frame_type}"
+channel = "${base.channel}"
+start = "${base.stop}"
+stop = "${base.test_stop}"
+sample_rate = "${base.sample_rate}"
+prior = "mlpe.injection.priors.sg_uniform"
+waveform = "mlpe.injection.waveforms.sine_gaussian_frequency"
+n_samples = 8
+waveform_duration = "${base.waveform_duration}"
+spacing = 100
+buffer = 10
+datadir = "${base.datadir}"
+logdir = "${base.logdir}"
+gaussian = false
+
+[tool.typeo.scripts.sample-with-bilby]
+request_cpus = 12
+n_live = 1000
+n_act = 5
+ifos = "${base.ifos}"
+bilby_duration = "${base.bilby_duration}"
+waveform = "mlpe.injection.waveforms.sine_gaussian_frequency"
+datadir = "${base.datadir}"
+writedir = "${base.datadir}"
+logdir = "${base.logdir}"
+accounting_group = "${LIGO_ACCOUNTING_GROUP}"
+channel = "${base.channel}"
+sample_rate = "${base.sample_rate}"
+prior = "${base.prior}"
 
-[tool.typeo.scripts.sample_testing_set]
+[tool.typeo.scripts.sample-with-flow]
```

Comment:
undo these changes as well. They will be done in the next patch for the sampling.